akka-javasdk/src/main/scala/akka/javasdk/impl/effect/ReplicationFilterImpl.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2021-2025 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk.impl.effect
+
+import java.util
+
+import scala.jdk.CollectionConverters._
+
+import akka.annotation.InternalApi
+import akka.runtime.sdk.spi.SpiEventSourcedEntity
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[javasdk] object ReplicationFilterImpl {
+  val empty: ReplicationFilterImpl = ReplicationFilterImpl(Set.empty, Set.empty)
+}
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[javasdk] final case class ReplicationFilterImpl(addRegions: Set[String], removeRegions: Set[String])
```

Comment:
nitpicking, but I think the names here should be `includedRegions` and `excludedRegions`